docs/rules/0203/output-only.md

```diff
@@ -67,10 +67,10 @@ Remember to also include an [aip.dev/not-precedent][] comment explaining why.
 message Book {
   string name = 1;
 
-  // Immutable. The title of the book.
+  // @OutputOnly
```

Comment:
s/@OutputOnly/Output only. ? Is `@OutputOnly` a new form of documenting "Output only."?

Reply:
No, it is a very old form, that we want to discourage. That's why it is in the "if you are suppressing the linter" section.